Projects.js

```diff
@@ -9,6 +9,29 @@ email.addEventListener('input', (event) => {
   }
 });
 
+// ////////////////// Local Storage //////////////////
+const form = document.getElementById('form');
+let formDataObj;
+form.addEventListener('submit', (event) => {
+  event.preventDefault();
+  const myFormData = new FormData(event.target);
+  formDataObj = {};
+  myFormData.forEach((value, key) => {
+    formDataObj[key] = value;
+  });
+  const formData = JSON.stringify(formDataObj);
+  localStorage.setItem('Data', formData);
+});
+
+const userData = JSON.parse(localStorage.getItem('Data'));
+const body = document.getElementById('body');
+body.onload = () => {
+  email.value = userData.email;
+  document.getElementById('text').value = userData.text;
+  document.getElementById('textarea').value = userData.message;
+};
```

Comment:
- [ ] Nice job so far 👍🏿 on your project, implementing the preserve data, but i notice when i fill out the form and click on the submit button the form is not submitting and it throws this error, i kindly suggest you check your code and fix it 😉 
- [ ] the error

![image](https://github.com/sheezu900/My_Potfolio/assets/68823331/64f63398-3bd2-4c15-bef5-6f25babf5e3d)